e2e/wdio/platform/pages/time-picker.po.ts

```diff
@@ -0,0 +1,43 @@
+import { BaseComponentPo } from './base-component.po';
+import { waitForElDisplayed, waitForPresent } from '../../driver/wdio';
+
+export class TimePickerPO extends BaseComponentPo {
+    url = '/time-picker';
+    root = '#page-content';
+    errorBorder = '.is-error';
+    setToNullButton = '[label="Set To Null"]';
+    setValidTimeButton = '[label="Set Valid Time"]';
+    timePickerInput = 'fd-time-picker input';
+    timePickerButton = 'fd-time-picker button';
+    activeTimePickerInput = 'fd-time-picker[ng-reflect-is-disabled="false"] input';
+    activeTimePickerButton = 'fd-time-picker[ng-reflect-is-disabled="false"] button';
+    timerExpanded = 'div.fd-time';
+    navigationDownArrowButton = 'button[glyph="navigation-down-arrow"]';
+    timeItem = 'span.fd-time__item';
+    selectedHours = '(//div[contains(@class, "fd-time__wrapper")]//li[contains(@class, "fd-time__item")])[12]';
+    selectedMinutes = '(//div[contains(@class, "fd-time__wrapper")]//li[contains(@class, "fd-time__item")])[54]';
+    period = '//span[contains(text(), " PM ")]/parent::li';
```

Comment:
try to not use the text 
'.fd-time__col--period span.fd-time__unit' and then 0 or 1